lambdatest/README.md

```diff
@@ -0,0 +1,42 @@
+# LambdatTest integration
+
+## Overview
+
+Easily integrate with LambdaTest and empower your teams to collaborate and test efficiently. LambdaTest is the fastest-growing cloud based testing platform that allows users to run both manual and automated tests on their websites and web apps across 2000+ browsers, browser versions, and operating systems. 
+
+The highly reliable and scalable test execution platform can be used to run both manual and automated tests and supports various automation testing frameworks like Selenium, Cypress, TestCafe, etc.
+
+With LambdaTest and Datadog integration, you can log bugs with a single click while performing cross-browser testing of your websites (and web apps) from the LambdaTest platform to Datadog. LambdaTest will automatically include details from the testing environment, such as browser version, OS, resolution, comments, along with screenshots.
+
+Here's all that you can do with LambdaTest:  
+- Live-interactive testing across 2000+ browsers & real machines hosted on cloud-based infrastructure.
+- An online automation testing grid supports Selenium & Cypress tests with any CI/CD pipeline to help QA teams validate and ship quality builds faster.
+- A next-gen developer-friendly browser that helps to build well-performing & responsive websites on the fly.
+- 100+ integrations with third-party tools for project management, communication, codeless automation, CI/CD, and more.
+- Help is always available with 24/7 chat support.
+- Lifetime free access to the platform with 100 free automation testing minutes.
+
+## Setup
+
+All configuration happens on Lambdatest Dashboard. Find details on how to set it up in the [LambdaTest-Datadog integration Documentation][3]
```

Comment:
```suggestion
All configuration happens on Lambdatest Dashboard. Find details on how to set it up in the [LambdaTest-Datadog integration Documentation][1]
```

Reply:
This integration does not submit any service checks, metrics or logs.